packages/webviz-core/src/dataProviders/MemoryCacheDataProvider.js

```diff
@@ -66,7 +69,8 @@ export function getBlocksToKeep({
     const blockRange = recentBlockRanges[blockRangeIndex];
 
     // Work backwards from the end of the range, since those blocks are most relevant to keep.
```

Comment:
Comment is out of date now?

Reply:
I guess it depends on how you interpret "the end" :thinking: